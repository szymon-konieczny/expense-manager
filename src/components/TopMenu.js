import * as React from 'react';
import { NavLink } from 'react-router-dom';

const TopMenu = () => (
    <nav>
        <ul>
            <li><NavLink to="/" activeClassName="is-active" exact={true}>Dashboard</NavLink></li>
            <li><NavLink to="/create" activeClassName="is-active">Create expense</NavLink></li>
            <li><NavLink to="/help" activeClassName="is-active">Help</NavLink></li>
        </ul>
    </nav>
);
export default TopMenu;