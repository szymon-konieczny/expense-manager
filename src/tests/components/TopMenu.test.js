import * as React from 'react';
import { shallow } from 'enzyme';
import TopMenu from '../../components/TopMenu';

test('should render TopMenu correctly', () => {
    const wrapper = shallow(<TopMenu />);
    expect(wrapper).toMatchSnapshot();
});