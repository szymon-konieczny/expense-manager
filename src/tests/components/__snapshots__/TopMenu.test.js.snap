// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TopMenu correctly 1`] = `
<nav>
  <ul>
    <li>
      <NavLink
        activeClassName="is-active"
        ariaCurrent="true"
        exact={true}
        to="/"
      >
        Dashboard
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="is-active"
        ariaCurrent="true"
        to="/create"
      >
        Create expense
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="is-active"
        ariaCurrent="true"
        to="/help"
      >
        Help
      </NavLink>
    </li>
  </ul>
</nav>
`;
